intro_calculating_cred_grain.md

```diff
@@ -0,0 +1,162 @@
+
+#### Manual input:
+
+It’s also worth noting that not all contributions occur on platforms with automated data fetching, and thus the capability exists to individually add them to the weighted graph directly at this stage. It’s clear that there will always be data that has to be incorporated manually, and how to incorporate those contributions, especially those not easily quantified, remains an active area of discussion.
+
+#### Merging graphs:
+
+A plugin will use the API for the collaborative workspace app or platform to generate a graph of nodes and edges. One interesting caveat is that there isn't necessarily one most-intuitively correct way to merge two weighted graphs if they share either edges or nodes (i.e. how would we calculate the combined weight). Currently, the graph nodes are nearly entirely disjoint, the weights are added together, and identities are collapsed into one.
+
+#### Summary up to this point:
+
+So at this point, we’ve got data from all our sources combined in one weighted, directed graph, where contributions and contributors are nodes, interactions and connections between nodes are edges, each have a relative weight based on parameters the community has set, and we’re ready to run PageRank on it.
+
+## SourceCred’s PageRank
+
+The next goal will be to generate an evaluation of the relative importance of nodes based on our weighted graph, specifically the weights of the nodes and edges.
+
+#### PageRank explained elsewhere
+
+Before reading on, it’s important to understand what PageRank does. There are other good explanations out there, and keeping this brief seems useful, so I won’t add another explanation. If you like over preparing, Brilliant.com’s explanation of [Markov Chains](https://brilliant.org/wiki/markov-chains/#transition-matrices), [Stationary Distributions](https://brilliant.org/wiki/stationary-distributions/) and [their quiz](https://brilliant.org/practice/basic-markov-chains) is more than sufficient. If you’re a visual thinker, [these slides](https://www.cl.cam.ac.uk/teaching/1516/InfoRtrv/lecture8-link-analysis-2x2.pdf) provide some nice figures.
+
+<img src="https://upload.wikimedia.org/wikipedia/commons/6/69/PageRank-hi-res.png" width="400">
+
+> Figure 4. An illustration of PageRank, in which the size of each node is proportional to the total size of the nodes linking to it.
+
+#### Stationary distribution intuition
+
+The simplest formulation of PageRank operates over an unweighted graph, treating all links as equivalent, as illustrated in the figure above. It assigns all outbound links equal probability and calculates the stationary distribution of node weights. (Intuitively, the stationary distribution is the equilibrium fraction of water that ends up in each node “tank” if you run a pump along each edge for a while – for “reasonable” graph configurations you’d expect the water distribution to end up the same regardless of where you dump the water initially.)
```

Comment:
Personally, I like the metaphor of a river system where water has ponds where it accumulates, and channels that link the ponds. At every pond, some of the water seeps into the earth, but it bubbles back up at "spring" (some special ponds have springs). 

This way you can get an intuition for the seed behavior.

Reply:
added!  (I like that much better than my pumps and pools, to which I was going to add some weird rain phenomenon)